packages/vscode-extension/src/types/ActionResult.d.ts

```diff
@@ -10,6 +10,15 @@ export interface ActionResult<T> {
 	data: T;
 }
 
+export interface AuthenticateActionResult extends ActionResult<undefined> {
```

Comment:
It's really undefined in ActionResult<undefined>?

Reply:
Yes, an AuthenticateActionResult doesn't have `data`, so it's an ActionResult with `data` property `undefined`